docs/api.md

```diff
@@ -203,9 +218,103 @@ assert response.status_code == 200
 ```
 
 
-### Report
+## Report Endpoint
+
+Two types of requests can be made:
+
+- To request a Report for an AOI defined by OQT provide following parameters: `name`, `dataset` and `featureId`
+- To request a Report for a custom AOI provide following parameters: parameters: `name` and `bpolys`
+
+Depending on the input, the output will be a GeoJSON Feature or FeatureCollection with the results of the Report and each Indicator as properties. The Feature properties of the input GeoJSON will be preserved if they do not collide with the properties set by OQT.
+
+### Parameters
+
+- `name`: Name of the Rerpot
+- `dataset`: Name of the dataset containing bounding polygons
+- `featureId`: Identifier of the feature in the dataset
+- `bpolys`: Bounding polygon(s) as GeoJSON Feature or FeatureCollection object. The Geometry of those objects has to be Polygon or MultiPolygon.
+- `includeSvg` (boolean): Include a SVG string of a figure displaying the Indicator
+results
+- `includeHtml` (boolean): Include a HTML string with the Indicator results
```

Comment:
See table comment above

Reply:
Yeah. A table is quite neat.
I changed to list to a table.